tests/integration/dogshell/test_dogshell.py

```diff
@@ -324,20 +327,21 @@ def test_monitors(self):
         out = json.loads(out)
         assert updated_message == out["message"]
         assert query == out['query']
-        assert monitor_name == out ['name']
+        assert monitor_name == out['name']
         assert current_options == out['options']
 
-        # Updating query only
+        # Updating optional type and query
         updated_query = "avg(last_15m):sum:system.net.bytes_rcvd{*} by {env} > 222"
+        updated_type = 'query alert'
 
         out, err, return_code = self.dogshell(
-            ["monitor", "update", monitor_id, "--query", updated_query]
+            ["monitor", "update", monitor_id, "--type", updated_type, "--query", updated_query]
         )
 
         out = json.loads(out)
         assert updated_query in out["query"]
```

Comment:
Let's also add an assert on the updated type

Reply:
oops yep sorry I overlooked this